tests/app/main/test_frameworks.py

```diff
@@ -390,7 +391,7 @@ def test_downloads_shown_open_framework(self, data_api_client, s3):
         )
         assert len(doc.xpath(
             "//main//p[contains(normalize-space(string()), $a)]",
-            a="until 5pm BST, 22 September 2015",
+            a="until 5pm BST, Tuesday 22 September 2015",
```

Comment:
Strange that the content has changed here. Was this just a oddity?

Reply:
Yeah, in G-Cloud 7 dates, we didn't show the name of the day. In later (current) frameworks, we do. Now that we're storing it as timestamps and formatting it consistently, all frameworks will show the day of the month. Looks weird, but it means our test cases more accurately represent the current content, so I think it's all good 👍

Reply:
https://github.com/alphagov/digitalmarketplace-frameworks/blob/e1f8e5c4a1a98d817e1bc9ff90ace85c1a0762c6/frameworks/g-cloud-7/messages/dates.yml

vs

https://github.com/alphagov/digitalmarketplace-frameworks/blob/e1f8e5c4a1a98d817e1bc9ff90ace85c1a0762c6/frameworks/g-cloud-10/messages/dates.yml